### DISCUSSION TITLE
request blocks for user details and preference settings [blocks]:

### DISCUSSION THREAD

**Discussion by darbinet** [shadcn-ui/ui#3791] [Blocks Request]
### Description

thank you for the great efforts you've put the make this project successful, I wish if you could please add blocks for user preferences and settings, that will be so much helpful.
thank you

### Example

https://preview.keenthemes.com/metronic8/demo1/account/overview.html

**Comment by tabarra** [comment]
Kinda covered by this block: https://ui.shadcn.com/blocks#sidebar-13, if you are willing to use a dialog.
